tests/app/data/test_schema_validation.py

```diff
@@ -28,11 +29,16 @@ def test_schema(self):
         for file in files:
 
             json_file = open(os.path.join(settings.EQ_SCHEMA_DIRECTORY, file), encoding="utf8")
+            json_to_validate = json.load(json_file)
+
+            error = self.validate_json_against_schema(file, json_to_validate, schema)
+
+            error = self.validate_schema_contains_valid_routing_rules(file, json_to_validate)
```

Comment:
Each time error is set is overriding the last before the list is extended finally below (will only lost last error found if there were multiple); maybe extend errors immediately? Or alternatively only call later checks of previous check had no errors?